cli/lib/kontena/callback.rb

```diff
@@ -22,14 +22,17 @@ def self.matches_commands(*commands)
       else
         cmd_type = cmd_type.to_sym
       end
-      cmd_types[cmd_class.to_sym] = cmd_type
+      cmd_types[cmd_class.to_sym] ||= []
```

Comment:
I have no clue why I made this double loop thing back then.

Anyways, the problem here was that without this fix, something like:
```
matches_commands '* exec', '* ssh'  
```
would only add the last one.